third_party/move/move-compiler-v2/tests/ability-check/loop_abort.exp

```diff
@@ -1,7 +1,7 @@
 
 Diagnostics:
 error: local `_x` of type `Test::Impotent` does not have the `drop` ability
-   ┌─ tests/ability-check/loop_abort.move:11:13
+   ┌─ tests/ability-check/loop_abort.move:11:18
    │
 11 │         let _x = Impotent {};
-   │             ^^ implicitly dropped here since it is no longer used
+   │                  ^^^^^^^^^^^ implicitly dropped here since it is no longer used
```

Comment:
I can't decide which was the better error output, and whether this may be related to https://github.com/aptos-labs/aptos-core/issues/11599.